project_event/wizard/double_task_validation_wiz.py

```diff
@@ -0,0 +1,38 @@
+# @ 2018 Savoir-failre Linux
+# License LGPL-3.0 or Later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class DoubleTaskValidationWiz(models.TransientModel):
+    """Wizard Reservation Confirmation"""
+    _name = 'double.task.validation.wiz'
```

Comment:
the class name is ambiguous: It seems like if you will do a double check of the task